rst2ctags.py

```diff
@@ -213,7 +215,10 @@ def main():
 
     for filename in args:
         f = open(filename, 'rb')
-        lines = f.read().splitlines()
+        try:
+            lines = f.read().decode('utf-8').splitlines()
+        except UnicodeDecodeError:
+            lines = f.read().splitlines()
```

Comment:
This does not work.  The example you demonstrated had all ascii characters, and is valid UTF-8.  If you insert an invalid UTF-8 character, you'll find that f.read() succeeded, moving the file pointer to the end of the file, and the decode step fails.  Now the exception kicks in, and you attempt to read the remainder of the file and get an empty string.  Perhaps something more like this should be done:

```python
        f = open(filename, 'rb')
        buf = f.read()

        try:
            buf = buf.decode('utf-8')
        except UnicodeDecodeError:
            pass

        lines = buf.splitlines()

        f.close()
        del buf
```